Misc/NEWS.d/next/Security/2025-01-14-11-19-07.gh-issue-128840.M1doZW.rst

```diff
@@ -0,0 +1 @@
+Limit the number of parts split during IPv6 address parsing.
```

Comment:
This describes what the code internally does. But this is not interesting to the user. The user is interested in the effect that this change have.

Now, I'm not sure what real issue this change fixes. It may reduce time and memory consumption for handling some invalid IPv6 addresses, but is it noticeable? Are there any measurements? Note that proportional memory already spent on the input string and  proportional time was already spent on reading and decoding it. There are usually limits on the total size of the user request and the length of the line.

So I would make general remark about improving the time and memory consumption when parsing certain invalid IPv6 addresses without much details.

Reply:
I've added a more descriptive newsfragment in https://github.com/python/cpython/pull/128841/commits/e65928735aae806d1343d7b14802fb6dd0fb6b64